### ISSUE TITLE
Graphs empty when using probe fping

### ISSUE THREAD

**Issue by TuxedoOne** [oetiker/SmokePing#119]
Hey,

when I try to use fping as a probe, the corresponding graphs are empty. Other probes work fine.
I did find out that when I start smokeping as root user fping is also working.
So I started smokping in debug mode under the user smokeping and come across the following error.

FPingGebA: Executing /usr/bin/fping -C 30 -q -B1 -r1 -t500 -i1 -p500 10.10.10.10
FPingGebA: Got fping output: '/usr/bin/fping: these options are too risky for mere mortals.'
FPingGebA: Got fping output: '/usr/bin/fping: You need i >= 10, p >= 20, r < 20, and t >= 50'

Thank you in advance.

**Comment by heistp** [comment]
fping must be setuid root (chmod u+s /usr/bin/fping).

**Comment by heistp** [comment]
You're right, fping doesn't have to be setuid root. I read the first sentence and responded too quickly as I had a problem in an lxc container for which that solved it, however I could have gotten it working a better way. Anyway, glad it works for you. 👍
